### PR TITLE
server: log typo fix.

### PR DESCRIPTION
**Pull request by qiuyesuifeng**

### CODE REVIEW

**Comment by shenli** [comment]
LGTM

**Comment by qiuyesuifeng** [comment]
@siddontang PTAL

File: server/balancer.go [hunk 1]
```diff
@@ -105,15 +105,21 @@ func (cb *capacityBalancer) SelectToStore(stores []*StoreInfo, excluded map[uint
...
 	if store == nil {
 		return nil, nil
 	}
 
 	// Random select one leader region from store.
 	storeID := store.store.GetId()
-	region := regions.randRegion(storeID)
+	region := cluster.regions.randRegion(storeID)
+
+	// If region peer count is less than max peer count, no need to do balance.
```
Comment (siddontang) [inline server/balancer.go]: do capacity balance.

Comment (siddontang) [inline server/balancer.go]: I think we should pass a filter in randRegion to filter these regions when selecting.

**Pull request merged**